engine/src/main/java/org/destinationsol/game/screens/MainScreen.java

```diff
@@ -253,6 +253,8 @@ public void updateCustom(SolApplication solApplication, SolInputManager.InputPoi
             if (!isOn) {
                 is.setOperations(is.chooseMercenary);
                 inputMan.addScreen(solApplication, is);
+                
+                game.getHero().getTradeContainer().getMercs().seenAll();
```

Comment:
`seenAll`
What is this supposed to mean? Rename it to something more meaningful.
(keep in mind the tense - `seenAll` sounds like a getter that returns if all entires are seen. You need `markAllAsSeen` or something like that)